src/client/cmd/launch.cpp

```diff
@@ -160,7 +161,27 @@ mp::ParseCode cmd::Launch::parse_args(mp::ArgParser* parser)
     {
         try
         {
-            auto node = YAML::LoadFile(parser->value(cloudInitOption).toStdString());
+            YAML::Node node;
+            const QString& cloudInitFile = parser->value(cloudInitOption);
+            if (cloudInitFile == "-")
+            {
+#ifdef MULTIPASS_PLATFORM_WINDOWS
```

Comment:
I prefer not to have any Windows specific code in `master`, so instead, we could add a new utility function definition in  `include/multipass/cli/client_platform.h` and then put the implementation in the appropriate file under `src/platform/client`.  For non-Windows platforms, it would just be a no-op.

Reply:
Done